components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -5700,6 +5701,15 @@ def _getQueryString(cls, opts=None):
             query += ' join obj.projectLinks plink'
             clauses.append('plink.parent.id = :pid')
             params.add('pid', rlong(opts['project']))
+        if opts is not None and 'orphaned' in opts and opts['orphaned']:
```

Comment:
Just as a matter of style, you could use the construct `opt.get('orphaned')` here.